pipeline/transformer/transformer_miro/src/main/scala/uk/ac/wellcome/platform/transformer/miro/transformers/MiroImage.scala

```diff
@@ -36,9 +20,6 @@ trait MiroImage {
         value = miroRecord.imageNumber
       ),
       version = version,
-      location = DigitalLocation(
-        url = buildImageApiURL(miroRecord.imageNumber, "info"),
-        locationType = LocationType("iiif-image")
-      )
+      location = getLocations(miroRecord).head
```

Comment:
Is there any case in which `getLocations()` could return multiple locations? And if not, why not have a `getLocation()` method instead?

Reply:
Yeah, I didn't want to mess around with this as at the moment it can only ever return 1 location, but it's probably clearer for all involved if I change it to `getLocation` in existing code too